test/fuzz/transformation_flatten_conditional_branch_test.cpp

```diff
@@ -2189,6 +2193,10 @@ TEST(TransformationFlattenConditionalBranchTest, ContainsSynonymCreation) {
                    .IsApplicable(context.get(), transformation_context));
 }
 
+// TODO: VULKAN_1_1
+
+// TODO: VULKAN_1_1 with 1.4
+
```

Comment:
Not sure if you forgot to address these or left them for later :)

Reply:
Well spotted - thanks! I had addressed them but forgot to remove them. Done now.